dev/reproducibility.py

```diff
@@ -43,11 +43,11 @@ def compute_csa(config, df_results, logdir, bids):
         # Get GT csa
         gt_path = os.path.join(bids, "derivatives", "labels", subject.split("_")[0],
                                "anat", subject + config["loader_parameters"]["target_suffix"][0] + ".nii.gz")
-        os.system(f"sct_process_segmentation  -i {gt_path} -append 1 -perslice 1 -o csa.csv")
+        os.system(f"sct_process_segmentation  -i {gt_path} -append 1 -perslice 1 -angle-corr 0 -o csa.csv")
```

Comment:
why explicitly using "append 1" if you delete the file afterwards? (it confuses code readers)

Reply:
Absolutely, it is unnecessary here. Updated here:  c2d8388